syntax = "proto3";

package member;

import "common.proto";

service MemberService {
    rpc StreamMembers (Empty) returns (stream MemberReply);
}

message MemberReply {
    string id = 1;
    string name = 2;
    int32 age = 3;
}
